src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -2276,7 +2268,10 @@ public async Task PostAsync_Redirect_LargePayload_Helper(Configuration.Http.Remo
                 // Compute MD5 of request body data. This will be verified by the server when it receives the request.
                 content.Headers.ContentMD5 = TestHelper.ComputeMD5Hash(contentBytes);
 
-                using (HttpClient client = CreateHttpClientForRemoteServer(remoteServer))
+                // Issue #39545: test occassionally timing on CI using default timeout of 100 seconds.
```

Comment:
Unless we're leaving the issue open, we shouldn't tag the issue like this. Otherwise it looks like it's an ActiveIssue. It's fine to link to the issue for more info, but we should comment something like:

"Because <x>, test can take very long to run, particularly in CI. See github issue <link> for more information."

That said, there isn't really any additional info in the github issue other than "this test takes a long time" so I'm not sure it's actually worth linking. The comment itself is pretty self explanatory.

Same comment below in the other test.

Reply:
Thanks, will adjust wording.